home-application/src/main/java/com/softserveinc/ita/homeproject/config/JerseyConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.softserveinc.ita.homeproject.config;
+
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider;
```

Comment:
Sorry for the inconvenience. I didn't notice (The same didn't you do). We have the generated one `import com.softserveinc.ita.homeproject.api.JacksonJsonProvider;` Please use it.

Reply:
Sure, I will try to use it